hash/msx1_cart.xml

```diff
@@ -25958,8 +25880,7 @@ legacy FM implementations cannot find it.
 	<software name="lostmine">
 		<description>The Lost Mines</description>
 		<year>2015</year>
-		<publisher>&lt;homebrew&gt;</publisher>
-		<info name="developer" value="Cesar Rincon"/>
+		<publisher>Cesar Rincon</publisher>
```

Comment:
In proper Spanish, it’s “César Rincón”, e.g. see here: https://www.generation-msx.nl/software/cesar-rincon/the-lost-mines/4135/